metashare/api/jobs.py

```diff
@@ -198,3 +198,10 @@ def delete_scratch_org(scratch_org):
 
 
 delete_scratch_org_job = job(delete_scratch_org)
+
+
+def refresh_github_repositories_for_user(user):
+    user.refresh_repositories()
```

Comment:
@wlonk This is also called from the `UserRefreshView` of `views.py` -- I assume that should also be converted to use the job instead.

Reply:
Thanks, yes, will do.